src/FluentValidation/Internal/Compatibility.cs

```diff
@@ -61,15 +61,21 @@ public static Func<string> CreateGetter(this PropertyInfo property)
 #endif
 		}
 
-		public static ValidatorAttribute GetValidatorAttribute(this Type type)
+		public static TAttribute GetCustomAttributeCompatible<TAttribute>(this Type type)

```

Comment:
@JeremySkinner .. I needed this to check for the existence of `FlagsAttribute`. I thought it would be better to expose this utility and encapsulate the portability aspects, rather than let that leak into `EnumValidator`.